android/app/src/main/AndroidManifest.xml

```diff
@@ -46,4 +46,5 @@
             android:value="2" />
     </application>
     <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.QUERY_ALL_PACKAGES"/>
```

Comment:
Any specific reason to use QUERY_ALL_PACLAGES permission over CAMERA ?

Reply:
yeah because camera works only for the sdk version below 30 and for above 30 we have to include this in the permissions